### ISSUE TITLE
Убрать границы у 2-го и 4-го столбца.

### ISSUE THREAD

**Issue by binexekb** [binexekb/ispublic#1]
Имеется таблица с 5-тю стобцами, нужно что бы у 2-го и 4-го столбцов не показывались границы слева и справо.

**Comment by komplekt17** [comment]
тебе походу нужно объединить ячейки?

**Comment by binexekb** [comment]
Нет. Запускал ХМЛ? Просто не показывать границы. у 2 и 4-го стобца.

**Comment by komplekt17** [comment]
а понятно, где стрелки, там не должно быть границ?

**Comment by binexekb** [comment]
Да. Визуально будет уютней.

**Comment by binexekb** [comment]
Только с шапкой не понятно. Где заголовок для стрелок... тоже что ли не отобржать. Или границу где-то поставить. Посреди.

**Comment by binexekb** [comment]
Наверное в заголовке таблицы убрать левые рамки для 2 и 4 столцов...

**Comment by komplekt17** [comment]
это тебе для тренировки или куда-то надо?

**Comment by binexekb** [comment]
[Сюда](http://infostart.ru/public/344684/)

**Comment by komplekt17** [comment]
у тебя по смыслу 3 столбца, а не 5?

**Comment by binexekb** [comment]
пять. Каждый столбец со смыслом. ))

**Comment by binexekb** [comment]
Подозреваю что для каждой строчки у тега ТД нужно будет писать "border-left-width: 0;"

**Comment by binexekb** [comment]
Внес изменения. Не смотрится. Видно прерывание границ.

**Comment by komplekt17** [comment]
вот так попробуй 
<table border="1">
	<thead>
			<th style="text-align: center;">Платформа EDI</th>
			<th style="border-left:none; border-right:none;"></th>
			<th style="text-align: center;">Агрегатор EDI</th>
			<th style="border-left:none; border-right:none;"></th>
			<th style="text-align: center;">Основная учетная система (ЦБ)</th>
	</thead>
	
		<tr>
			<td style="text-align: center;">ORDERS</td>
			<td style="border-left:none; border-right:none;"><img src="http://www.logointerpreter.com/images/right_arrow.png"/></td>
			<td><strong>Документ "Заказ клиента"</strong><br />
			Идентификация номенклатуры заказчика, контрагента, грузополучателя, адреса доставки.<br />
			Настройки:
			<ul>
				<li>Загружать заказы (ORDERS) - Загружать или игнорировать сообщения ORDERS.</li>
				<li>Отправлять заказы в ЦБ - Отправлять или нет заказ в ЦБ.</li>
				<li>Выгружать цены в ЦБ - Выгрузка документа в ЦБ с ценами или только с количеством.</li>
				<li>Разрешить пустые строки в заказах - Отправлять заказ в ЦБ, даже если товар не идентифицирован.</li>
			</ul>
			</td>
			<td style="border-left:none; border-right:none;"><img src="http://www.logointerpreter.com/images/right_arrow.png"/></td>
			<td><strong>Документ "Заказ клиента"</strong><br />
			Загрузка или обновление не проведенного документа. Заполнение цен, расчет сумм. Авторегистрация в плане обмена для выгрузки в агрегатор.</td>
		</tr>
		<tr>
			<td style="text-align: center;">ORDRSP</td>
			<td style="border-left:none; border-right:none;"><img src="http://www.logointerpreter.com/images/left_arrow.png"/></td>
			<td><strong>Документ "Заказ клиента"</strong><br />
			Обновление данных заказа. Если заказ согласован и для заказчика включена настройка "Отвечать на заказы (ORDRSP)", то формируется и отправляется сообщение ORDRSP.</td>
			<td style="border-left:none; border-right:none;"><img src="http://www.logointerpreter.com/images/left_arrow.png"/></td>
			<td><strong>Документ "Заказ клиента"</strong><br />
			Пользователь согласовывает дату и объем отгрузки, проводит заказ. Авторегистрация в плане обмена для выгрузки в агрегатор.</td>
		</tr>
		<tr>
			<td style="text-align: center;">DESADV</td>
			<td style="border-left:none; border-right:none;"><img src="http://www.logointerpreter.com/images/left_arrow.png"/></td>
			<td><strong>Документ "Черновик накладной"</strong><br />
			Загрузка данных о поставке. Идентификация документа основания. Проверка соответствия условиям поставки. Формирование и отправка сообщения DESADV заказчику.<br />
			Настройки:
			<ul>
				<li>Уведомлять об отгрузках (DESADV) - Уведомлять или нет заказчика об отгрузке.</li>
				<li>Ограничить превышение весового товара при отгрузке - Процент допустимого превышения заказанного весового товара при отгрузке.</li>
				<li>Ограничить превышение штучного товара при отгрузке - Процент допустимого превышения заказанного штучного товара при отгрузке.</li>
				<li>Создавать INVOIC на основании DESADV - Формирование и отправка сообщения INVOICE.</li>
			</ul>
			</td>
			<td style="border-left:none; border-right:none;"><img src="http://www.logointerpreter.com/images/left_arrow.png"/></td>
			<td><strong>Документ "Реализация товаров и услуг"</strong><br />
			Пользователь на основании заказа создает и проводит реализации. Авторегистрация в плане обмена для выгрузки в агрегатор.</td>
		</tr>
		<tr>
			<td style="text-align: center;">RECADV</td>
			<td style="border-left:none; border-right:none;"><img src="http://www.logointerpreter.com/images/right_arrow.png"/></td>
			<td><strong>Документ "Накладная"</strong><br />
			Загрузка данных о принятой поставке. Идентификация документа-основания поставки. Проверка соответствия допустимым условиям расхождений.<br />
			Настройки:
			<ul>
				<li>Загружать акты приёмки (RECADV) - Загружать или игнорировать сообщения RECADV.</li>
				<li>Допустимый процент потерь - Процент допустимого расхождения общего количества товара между отгрузкой и приемкой.</li>
				<li>Разрешить излишки в накладных - Не блокировать документы, в которых заказчик принял товар, который ему не оформляли в сопроводительных документах.</li>
				<li>Выгружать цены в ЦБ - Выгрузка документа в ЦБ с ценами или только с количеством.</li>
			</ul>
			</td>
			<td style="border-left:none; border-right:none;"><img src="http://www.logointerpreter.com/images/right_arrow.png"/></td>
			<td><strong>Документ "Реализация товаров и услуг"</strong><br />
			Обновление данных документа, перерасчет сумм документа. Авторегистрация в плане обмена для выгрузки в агрегатор.</td>
		</tr>
		<tr>
			<td style="text-align: center;">INVOICE</td>
			<td style="border-left:none; border-right:none;"><img src="http://www.logointerpreter.com/images/left_arrow.png"/></td>
			<td><strong>Документ "Накладная"</strong><br />
			Обновление данных о рассчитанных суммах. Если для заказчика включена настройка "Создавать INVOIC на основании RECADV", то формируется и отправляется сообщение INVOICE</td>
			<td style="border-left:none; border-right:none;"><img src="http://www.logointerpreter.com/images/left_arrow.png"/></td>
			<td><strong>Документ "Реализация товаров и услуг"</strong></td>
		</tr>
</table>

**Comment by komplekt17** [comment]
у тебя на том сайте, куда ты втыкаешь html есть свой редактор, похоже он не корректно вставляет код

**Comment by binexekb** [comment]
Есть такое. Только там можно вставлять ХТМЛ. Я думаю да, лучше сделать три колонки, а стрелки лепить в начале текста. Только что бы текст под стрелски не лез. Как это делается?

**Comment by binexekb** [comment]
Или это сложней чем рамки скрыть?

**Comment by komplekt17** [comment]
он залезет, это надо править стилями, или редактором на сайте

**Comment by komplekt17** [comment]
рамки сделать проще

**Comment by komplekt17** [comment]
ты вставь мой код, что получается?

**Comment by binexekb** [comment]
Ладн. Сейчас попробую на сайте поредактировать. Вот что из твоего получается. В шапке я сам правил.
![image](https://cloud.githubusercontent.com/assets/21037564/22888495/bb77c28a-f227-11e6-8bf5-a82375b3c51e.png)

**Comment by binexekb** [comment]
По сути ты для каждого ТД прописываешь. Такое же нужно прописать и для 1 и для 3 и для 5 колонки.

**Comment by komplekt17** [comment]
да, но это нужно прописывать в стилях

**Comment by binexekb** [comment]
Как это со стилями делается?

**Comment by komplekt17** [comment]
шапка таблицы такая должна быть 
```
<table border="1">
	<thead>
			<th style="text-align: center;">Платформа EDI</th>
			<th style="border-left:none; border-right:none;"></th>
			<th style="text-align: center;">Агрегатор EDI</th>
			<th style="border-left:none; border-right:none;"></th>
			<th style="text-align: center;">Основная учетная система (ЦБ)</th>
	</thead>
```

**Comment by komplekt17** [comment]
далее в том <td>, где не нужны рамки по сторонам, проставляешь style="border-left:none; border-right:none;"

**Comment by komplekt17** [comment]
далее в том ,`<td>`

**Comment by binexekb** [comment]
Ок. Ща накалдую.

**Comment by komplekt17** [comment]
ладно, я отползаю, не разберёшься, пиши
